server/events/terraform/terraform_client.go

```diff
@@ -442,11 +444,15 @@ func (c *DefaultClient) RunCommandAsync(ctx models.ProjectCommandContext, path s
 
 		// Asynchronously copy from stdout/err to outCh.
 		go func() {
-			s := bufio.NewScanner(stdout)
-			for s.Scan() {
-				message := s.Text()
-				outCh <- Line{Line: message}
-				c.projectCmdOutputHandler.Send(ctx, message)
+			// Don't stream terraform show output to outCh
+			cmds := strings.Split(tfCmd, " ")
```

Comment:
Would it be better to actually only allow plan and apply specifically instead of just having a deny list?

Upstream has support for using `terraform version`. and we might want to eventually support `terraform import`.

Reply:
Yeah I think it makes sense to have an allow list for better control over what's being streamed to the web UI. I've added a `LogStreamingValidCmds` with `init`, `plan`, and `apply` commands for now. We can add more commands as we go :)